libraries/csharp-dotnet-core2/Dockerfile

```diff
@@ -2,22 +2,18 @@ RUN apt-get -y update && \
     apt-get install --no-install-recommends -y \
         build-essential \
         ca-certificates \
-        checkinstall \
-        libbz2-dev \
-        libc6-dev \
-        libffi-dev \
-        libgdbm-dev \
-        libncursesw5-dev \
-        libreadline-gplv2-dev \
-        libsqlite3-dev \
         libssl-dev \
-        tk-dev \
-        uuid-dev \
         wget
 
-RUN wget -q https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb
+# Install .NET Core SDK
+ENV DOTNET_SDK_VERSION 2.2
+ENV UBUNTU_VERSION 16.04
+
+RUN wget -q https://packages.microsoft.com/config/ubuntu/$UBUNTU_VERSION/packages-microsoft-prod.deb
 RUN dpkg -i packages-microsoft-prod.deb
 
 RUN apt-get install -y apt-transport-https
 RUN apt-get update
-RUN apt-get install -y dotnet-sdk-2.2 
+RUN apt-get install -y dotnet-sdk-$DOTNET_SDK_VERSION
+
+RUN dotnet help
```

Comment:
don't run a help command here

Reply:
I think thats a trick to get dotnet to download and build itself. @zeryx ?

Reply:
yeah that actually initializes dotnet, otherwise it'll take a bunch of time during algo execution to conform. It's from the microsoft dotnet sdk dockerfile

Reply:
can you include this as a link in the dockerfile and include any license file as necessary then? like https://github.com/algorithmiaio/langpacks/blob/master/libraries/python36/Dockerfile#L1

Reply:
yea definitely can do.

Reply:
Should be fixed, what do you think @pmcq ?

Reply:
can you add  `&& rm -rf /var/lib/apt/lists/*` to it too like the first apt-get has

Reply:
check above ^